### ISSUE TITLE
Relu fake quant after Quantification Aware Training 

### ISSUE THREAD

**Issue by xiaomsky** [tensorflow/tensorflow#49372]
hi

when  do  Quantification Aware Training with tensorflow-model-optimization, for Relu op, 
after QAT, and found the min value of fake quant node is not zero ,and a constant value.

so what reason cause that , thanks.

**Comment by xiaomsky** [comment]
TF version is TF2.3.0.

**Comment by Saduf2019** [comment]
@xiaomsky 
Your request is not clear, can you explain in brief the issue faced, we see you haven filled the issue template as well.

**Comment by xiaomsky** [comment]
@Saduf2019
now I use quantize_model or quantize_annotate_layer to do QAT for test model , 
but I found after do quantization ,  the min value of Relu op fake_quant node is not zero, and is it right ?
also every relu op fake_quant node min value is the same constant value.
will upload simple code followed.
please help to check. thanks.

//////////////////////////////////////////
import tensorflow as tf
import numpy as np
print(f"tf verion = {tf.__version__}")

import tensorflow_model_optimization as tfmot
from tensorflow.keras.layers import Conv2D,MaxPool2D,Flatten,Dense,Dropout,ReLU
from tensorflow.python.framework.convert_to_constants import convert_variables_to_constants_v2
# load data
mnist = tf.keras.datasets.mnist

(x_train, y_train), (x_test, y_test) = mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0

x_train = x_train[..., tf.newaxis]
x_test = x_test[..., tf.newaxis]

t_img =x_test[0]
t_img = np.expand_dims(t_img, axis=0)
print('t_img shape:',t_img.shape)
print('t_img dtype:',t_img.dtype)


model = tf.keras.models.Sequential([
        Conv2D(filters=6,kernel_size=5,strides=(1,1),padding='same',use_bias=False,input_shape=(28,28,1)),
        ReLU(max_value=6,negative_slope=0,threshold=0),
        MaxPool2D(pool_size=(3,3),strides=2,padding="same"),
        Conv2D(filters=16,kernel_size=5,strides=(1,1),padding='same',use_bias=False),
        MaxPool2D(pool_size=(3,3),strides=2,padding="same"),
        Flatten(input_shape=(7, 7)),
        Dense(120),
        Dense(84),
        Dropout(0.2),
        Dense(10, activation='softmax')
    ])

def model_run():
    '''
        float32 model
    '''
    print("float32 model:")
    model.compile(optimizer='adam',
                    loss='sparse_categorical_crossentropy',
                    metrics=['accuracy'])
    print("==> training")
    model.fit(x_train, y_train, epochs=1)
    print("==> evaluate")
    model.evaluate(x_test,  y_test, verbose=2)

def quant_model_run():
    '''
        quantized model
    '''
    print("quantized model:")
    quant_model = tfmot.quantization.keras.quantize_model(model)

    quant_model.compile(optimizer='adam',
                    loss='sparse_categorical_crossentropy',
                    metrics=['accuracy'])
    print("==> training")
    quant_model.fit(x_train, y_train, epochs=1)
    print("==> evaluate")
    quant_model.evaluate(x_test,  y_test, verbose=2)
    
    if 1:     
        #save models
        full_model = tf.function(lambda Input: quant_model(Input))
        full_model = full_model.get_concrete_function(tf.TensorSpec(t_img.shape, t_img.dtype))
        frozen_func = convert_variables_to_constants_v2(full_model)
        frozen_func.graph.as_graph_def()
        layers = [op.name for op in frozen_func.graph.get_operations()]
        # Save frozen graph from frozen ConcreteFunction to hard drive
        tf.io.write_graph(graph_or_graph_def=frozen_func.graph,
                          logdir="./",
                          name="test_quant.pb",
                          as_text=False)
        
    
    

if __name__=='__main__':
    model_run()
    quant_model_run()
